singolo/index.html

```diff
@@ -0,0 +1,173 @@
+                </li>
+                <li>
+                	<span class="li_heart"></span>
+                    <h4>Love Is The Answer</h4>
+                    <p>Nulla sed nunc et tortor luctus faucibus. Morbi at aliquet turpis, et consequat felis. Quisque condimentum.</p>
+                </li>
+                <li>
+                	<span class="li_phone"></span>
+                    <h4>Responsive Layout</h4>
+                    <p>Sed porttitor placerat rhoncus. In at nunc tellus. Maecenas blandit nunc ligula. Praesent elit leo.</p>
+                </li>
+                <li>
+                	<span class="li_bubble"></span>
+                    <h4>24 / 7 Support</h4>
+                    <p>Vivamus vel quam lacinia, tincidunt dui non, vehicula nisi. Nulla a sem erat. Pellentesque egestas venenatis lorem .</p>
+                </li>
+                <li>
+                	<span class="li_star"></span>
+                    <h4>Feel Like A Star</h4>
+                    <p>Quisque hendrerit purus dapibus, ornare nibh vitae, viverra nibh. Fusce vitae aliquam tellus.</p>
+                </li>
+            </ul>
+        </div>
+    </div>
+	<div id="portfolio">
+        <div class="inner">
+        	<h2>Portfolio</h2>
+            <h3>Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.</h3>
+            <ul class="portfolio-menu filter">
+            	<li class="current all"><a href="#">All</a></li>
+                <li class="webdesign"><a href="#">Web Design</a></li>
+                <li class="graphicdesign"><a href="#">Graphic Design</a></li>
+                <li class="artwork"><a href="#">Artwork</a></li>
+            </ul>
+            <ul class="portfolio">
+            	<li>
+                    <a href="https://sun1.dataix-by-minsk.userapi.com/c637817/v637817249/27297/d1Amzl6nJ7g.jpg" rel="prettyPhoto[portfolio]">
+                        <img src="/singolo/images/img1.jpg" width="220" height="187" alt="#" />
+                    </a>
+                </li>
+            	<li>
+                    <a href="https://sun1.dataix-by-minsk.userapi.com/c637817/v637817249/27297/d1Amzl6nJ7g.jpg" rel="prettyPhoto[portfolio]">
+                        <img src="/singolo/images/img2.jpg" width="220" height="187" alt="#" />
+                    </a>
+                </li>
+            	<li>
+                    <a href="https://sun1.dataix-by-minsk.userapi.com/c637817/v637817249/27297/d1Amzl6nJ7g.jpg" rel="prettyPhoto[portfolio]">
+                        <img src="/singolo/images/img3.jpg" width="220" height="187" alt="#" />
+                    </a>
+                </li>
+            	<li>
+                    <a href="https://sun1.dataix-by-minsk.userapi.com/c637817/v637817249/27297/d1Amzl6nJ7g.jpg" rel="prettyPhoto[portfolio]">
+                        <img src="/singolo/images/img4.jpg" width="220" height="187" alt="#" />
```

Comment:
You have width="220" height="187" repeated. It's better to create a class.
And it's better to write something meaningful it alt="#", as in case screean reader is used, then url will be read.

Reply:
fixed